frontend/src/components/dashboard/Dashboard.jsx

```diff
@@ -57,20 +61,91 @@ export default function Dashboard() {
   // page title
   useTitle("IntelOwl | Dashboard", { restoreOnUnmount: true });
 
+  const { organization } = useOrganizationStore(
+    React.useCallback(
+      (state) => ({
+        organization: state.organization,
+      }),
+      [],
+    ),
+  );
+
   return (
     <Container fluid id="Dashboard">
-      <div className="g-0 d-flex align-items-baseline flex-column flex-lg-row mb-2">
+      {/* <div className="g-0 d-flex align-items-baseline flex-column flex-lg-row mb-2">
         <h3 className="fw-bold" id="Dashboard_title">
           Dashboard
         </h3>
+        <ButtonGroup className="ms-2 mb-3 d-flex align-items-center">
+          <Button
+            outline={!state}
+            color={state ? "primary" : "tertiary"}
+            onClick={() => handleChange()}
+          >
+            GLOBAL
+          </Button>
+          <Button
+            outline={state}
+            color={!state ? "primary" : "tertiary"}
+            onClick={() => handleChange()}
+          >
+              ORG
+          </Button>
+        </ButtonGroup>
         <ElasticTimePicker
           className="ms-auto"
           size="sm"
           defaultSelected={range}
           onChange={onTimeIntervalChange}
           id="Dashboard_timepicker"
         />
+
+      </div> */}
+
+      <div className="d-flex flex-wrap justify-content-between align-items-baseline mb-2">
+        <div>
+          <h3 className="fw-bold" id="Dashboard_title">
+            Dashboard
+          </h3>
+        </div>
+        <div className="d-flex flex-wrap align-items-baseline ">
+          {organization?.name ? (
+            <ButtonGroup className="mb-3">
+              <Button
+                outline={orgstate}
+                color={!orgstate ? "primary" : "tertiary"}
```

Comment:
I suggest to reverse the check, i think it's easier to understand:
`orgstate ? "tertiary" : "primary"`